pkg/builder/webhook.go

```diff
@@ -65,6 +66,12 @@ func (blder *WebhookBuilder) For(apiType runtime.Object) *WebhookBuilder {
 	return blder
 }
 
+// WithMutatorFactory takes an admission.HandlerFactory, a MutatingWebhook will be wired for the handler that this factory creates.
+func (blder *WebhookBuilder) WithMutatorFactory(factory admission.HandlerFactory) *WebhookBuilder {
+	blder.mutatorFactory = factory
+	return blder
+}
```

Comment:
Adding this makes the `WithDefaulter` a noop, which doesn't return an error and it's a misleading UX

Reply:
Since this function doesn't return an error, I'll leave it to the `Complete` function to report the error. Unless you are ok with a panic?